build.py

```diff
@@ -667,6 +692,12 @@ def create_dockerfile_linux(ddir, dockerfile_name, argmap, backends, repoagents,
 # Top-level include/core not copied so --chown does not set it correctly,
 # so explicit set on all of include
 RUN chown -R triton-server:triton-server include
+'''
+
+    df += install_dcgm_libraries()
+    df += fail_if_dcgm_api_export_exists()
```

Comment:
Don't need to call 'fail...' here. We just want the libraries installed... not even going to use the includes.